nb-engine/nb-engine-cli/src/main/java/io/nosqlbench/engine/cli/NBCLI.java

```diff
@@ -76,8 +76,10 @@ public class NBCLI implements Function<String[], Integer>, NBLabeledElement {
     private static final int EXIT_OK = 0;
     private static final int EXIT_WARNING = 1;
     private static final int EXIT_ERROR = 2;
+    private static final String version;
 
     static {
+        version = new VersionInfo().getVersion();
```

Comment:
This needs to be moved below the logger config. The logger config is very sensitive to initialization order since hoisting any lo4j2 via the class loader can cause it to auto-config outside of the intended settings below.